File: problems-1-to-100/problem-1/problem-1.py
def multiples_three_and_five(number: int) -> int:
    sum = 0

    for n in range(3, number):
        if n % 3 == 0 or n % 5 == 0:
            sum += n

    return sum


multiples_three_and_five(1000)  # 233168

File: problems-1-to-100/problem-1/problem-1.rb
def multiples_three_and_five(number)
  sum = 0

  (3...number).each do |n|
    if n % 3 == 0 || n % 5 == 0
      sum += n
    end
  end

  sum
end

multiples_three_and_five(1000) # 233168
